Bridg-It.depend

```diff
@@ -0,0 +1,6 @@
+# depslib dependency file v1.0

```

Comment:
Chestia asta n-ar trebui sa existe. este bagata in .gitignore, deci probabil i-ai dat tu add, nu stiu. Anyway, nu o urcam pe master.